### PR TITLE
Add May blog post

### PR DESCRIPTION
**Pull request by dholbach**
Note I'm off for the rest of the week. If you feel like making changes to the post, please do so.

I wasn't quite sure about the meeting days suggested at the end of the post, will we cancel the meeting next week?

### CODE REVIEW

**Comment by stefanprodan** [comment]
> will we cancel the meeting next week?

Yes, there is too much going on at KubeCon.

**Review by stefanprodan — APPROVED** [review]

**Comment by scottrigby** [comment]
Pushed changed discussed with @stefanprodan 🆗

**Comment by staceypotter** [comment]
LGTM 👍

File: content/en/blog/2021-04-29-may-update.md [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+
+We feel very fortunate to have Alison on board!
+
+### Over and out
+
+If you like what you read and would like to get involved, here are a few
+good ways to do that:
+
+- Join our [upcoming dev meetings](https://fluxcd.io/community/#meetings) on
+  2021-05-06 15:00 UTC, or 2021-05-12, 12:00 UTC
```
Comment (dholbach) [inline content/en/blog/2021-04-29-may-update.md:232]: The meeting times need changing then, right, if we skip one meeting?

**Pull request merged**